Firestore/core/src/util/config.h

```diff
@@ -1,57 +0,0 @@
-/*
```

Comment:
This file shouldn't be deleted (unless there's a replacement?).

Note that config.h was previously generated, but this was changed for compatibility with Swift Package Manager which does not allow for build-time configuration scripts.

Reply:
Sorry, originally I deleted the wrong `config.h` file. Thanks for explaining!